cumulusci/tasks/salesforce/tests/test_enable_prediction.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+from unittest.mock import MagicMock
+from simple_salesforce import SalesforceError
+from cumulusci.tasks.salesforce.enable_prediction import EnablePrediction
+from cumulusci.core.exceptions import SalesforceException
+from .util import create_task
+
+
+class TestEnablePrediction(unittest.TestCase):
+    def setUp(self):
+        self.task = create_task(
+            EnablePrediction, {"developer_name": "mlpd__test_prediction_v0"}
+        )
+        self.task.tooling = MagicMock()
+
+    def test_run_task(self):
```

Comment:
This test is fine, but using `responses` to validate the HTTP calls can be cleaner, without needing as much mocking.